iati/core/resources/test_data/202/valid_dateorder.xml

```diff
@@ -1,15 +1,65 @@
 <?xml version="1.0"?>
-
+<!--dates in correct order and correctly formatted-->
 <root_element>
-  <!--empty path strings given-->
-  <element1>2000-12-01</element1><!--first element is in the correct date order-->
-  <element2>2000-12-24</element2><!--first element is in the correct date order-->
-  <element3>2017-08-04</element3><!--more is NOW-->
+  <condition></condition><!--condition case-->
+  <element1>2016-08-16</element1><!--first element is in the correct date order-->
+  <element2>2017-08-16</element2>
+  <element3>2016-08-16</element3><!--more is NOW-->
   <element4>9999-12-01</element4><!--less is NOW-->
-  <element5>2016-01-01</element5><!--duplicate path strings given-->
-  <element6 attribute="2014-01-01"></element6><!--Attribute dates are in the correct order-->
-  <element7 attribute="2014-12-31"></element7><!--Attribute dates are in the correct order-->
-  <element8 attribute="2017-01-01"></element8><!--more attribute is NOW-->
-  <element9 attribute="9999-12-01"></element9><!--less attribute is NOW-->
-  <element10 attribute="2015-04-03"></element10><!--duplicate attribute path given-->
+  <element5>2016-08-16</element5><!--multiple identical `less` values-->
+  <element5>2016-08-16</element5>
+  <element6>2017-08-16</element6>
+  <element7>2016-08-16</element7><!-- multiple identical `more` values-->
+  <element8>2017-08-16</element8>
+  <element8>2017-08-16</element8>
+  <nest>
+    <element9>2016-08-16</element9><!--nested elements-->
+    <element10>2017-08-16</element10>
+  </nest>
+  <nOw>2016-08-16</nOw>
```

Comment:
Missing comment to detail what this test is

Reply:
Same for element-based NOw

Reply:
done